File: String/Reverse String.cs
public class Solution {
    public void ReverseString(char[] s) {
        char temp;

        for (int i = 0; i < s.Length/2; i++)
        {
            temp = s[i];
            s[i] = s[s.Length-(i+1)];
            s[s.Length-(i+1)] = temp;
        }
    }
}

File: String/ReverseString.cpp
#include<iostream>
#include<vector>
using namespace std;

class Solution {
public:
    void reverseString(vector<char>& s) {
        int start = 0;
        int end = s.size()-1;
        while (start < end) {
            char temp = s[start];
            s[start++] = s[end];
            s[end--] = temp;
        }
    }
};

int main () {
    Solution* obj;
    vector<char> test = {'h', 'e', 'l', 'l', 'o'};
    obj->reverseString(test);

    for (int i = 0; i < test.size(); i++)
    {
        cout << test[i];
    }
    
}